contact_map/concurrence.py

```diff
@@ -0,0 +1,246 @@
+import itertools
+import mdtraj as md
+import numpy as np
+
+try:
+    import matplotlib.pyplot as plt
+except ImportError:
+    HAS_MATPLOTLIB = False
+else:
+    HAS_MATPLOTLIB = True
+
+
+class Concurrence(object):
+    """Superclass for contact concurrence objects.
+
+    Contact concurrences measure what contacts occur simultaneously in a
+    trajectory. When defining states, one usually wants to characterize
+    based on multiple contacts that are made simultaneously; contact
+    concurrences makes it easier to identify those.
+
+    Parameters
+    ----------
+        values : list of list of bool
+            the whether a contact is present for each contact pair at each
+            point in time; outer list is length number of frames, inner list
```

Comment:
this is the other way around outer list is length contact pair and inner list is frame